apps/client/src/config.ts

```diff
@@ -1,16 +1,30 @@
-export const isProduction =
+type Topic = {
+  slug: string;
```

Comment:
This is great!

Question about these props though. How are they ordered?
I think it would be nice to have all non-optional (`?`) props listed at the top. What do you think? Could the ordre maybe even be enforced by eslint/prettier?

Reply:
I ordered them by non-optional props on top,
Than optional props, and after that prop that are types.